collect_app/src/main/java/org/odk/collect/android/formentry/QuitFormDialog.java

```diff
@@ -0,0 +1,72 @@
+package org.odk.collect.android.formentry;
+
+import android.content.Context;
+import android.widget.ListView;
+
+import androidx.appcompat.app.AlertDialog;
+
+import com.google.common.collect.ImmutableList;
+
+import org.odk.collect.android.R;
+import org.odk.collect.android.adapters.IconMenuListAdapter;
+import org.odk.collect.android.adapters.model.IconMenuItem;
+import org.odk.collect.android.logic.FormController;
+import org.odk.collect.android.preferences.AdminKeys;
+import org.odk.collect.android.preferences.AdminSharedPreferences;
+import org.odk.collect.android.utilities.DialogUtils;
+
+import java.util.List;
+
+public class QuitFormDialog {
+
+    private QuitFormDialog() {
+    }
+
+    public static AlertDialog show(Context context, FormController formController, Listener listener) {
+        String title;
+        {
```

Comment:
Why did you add this block?

Reply:
Ahh ok it was there, but still we could get rid of it.

Reply:
Yeah I didn't notice that somehow!